docs/_includes/toc-intro.adoc

```diff
@@ -5,11 +5,13 @@ Included in:
 ////
 
 A table of contents (TOC) is a list of section and nested subsection titles that is generated when Asciidoctor processes a document.
-The TOC is inserted directly below the document's title and author, titled _Table of Contents_, and lists the section 1 and section 2 level titles by default.
 
-When the `docbook` converter is used, a TOC is inserted by default.
-To include a TOC when using the `html` or a custom backend, you must set the `toc` attribute.
-The toc attribute can be specified via the command line (`-a toc`) or in the document's header (`:toc:`)
+To include a TOC, you must set the `toc` attribute.
+
+[NOTE]
+This section applies to the html and pdf backends only. `toc` is ignored by the docbook backend because the TOC in docbook is controlled by the docbook stylesheets.
```

Comment:
The `toc` is not completely ignored by the DocBook converter. It's just a switch. It is either set or not set. The placement is what is ignored.